example/samples/printing_samples/ReceiptSamples/ReceiptSample06_PharmacyStore2_FieldData.json

```
@@ -0,0 +1,61 @@
+{
+    "address" : "123 Star Road, City,\nState 12345",
+    "telephone_number" : "012-345-67890",
+    "datetime" : "01/24/2023 12:43 PM",
+    "staff_name" : "John Smith",
+    "rfn" : "0913-8279-2597-0104-2420",
```

Biome (2.1.2)

[error] The key rfn was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)